.environment/gitleaks/run-gitleaks.sh

```diff
@@ -62,25 +64,25 @@ fi
 function base_command() {
     if [[ ${LOCAL_GITLEAKS?} == 1 ]]; then
         echo "running gitleaks locally"
-        gitleaks \
-            --path="${REPO_ROOT?}" \
-            --repo-config-path="${REPO_CONFIG_PATH?}" \
-            --report="${REPO_ROOT?}/${REPORT_JSON?}" \
+        gitleaks protect \
+            --source="${REPO_ROOT?}" \
+            --report-path="${REPO_ROOT?}/${REPORT_JSON?}" \
+            --config="${SCRIPT_DIR?}/${CONFIG_FILE?}" \
+            --report-format "json" \
             $(if [[ ${VERBOSE?} != 0 ]]; then echo "--verbose"; else echo ""; fi) \
-            $1 \
-            2>"${LOGFILE?}"
+            $1 
```

Comment:
I think setting to terminal output is fine, but I hope others agree so that it doesn't hold up the bug fix :/

Reply:
Agreed on terminal output -- it's much easier for humans to deal with.

Reply:
Yup. thx